components/ord-service/src/main/java/com/sap/cloud/cmp/ord/service/storage/model/helperentities/EntityTypeTarget.java

```diff
@@ -0,0 +1,15 @@
+package com.sap.cloud.cmp.ord.service.storage.model.helperentities;
```

Comment:
Move this file to `package com.sap.cloud.cmp.ord.service.storage.model;` (on the same level as `EntityTypeMappingEntity`)